server/src/main/java/io/crate/planner/operators/Collect.java

```diff
@@ -193,7 +196,7 @@ public List<Symbol> outputs() {
     }
 
     public WhereClause where() {
```

Comment:
should we maybe change the name to reflect it's the immutable `where`?

Reply:
Not sure. It's kinda an implementation detail so I'd tend to not expose it. If we start doing it for one property, then there's the question why other public properties don't have an immutable prefix.

The cases where the internal variables are accessed directly are an exception and ideally we try to avoid doing that more than necessary.